src/o365/spo/commands/site/site-list-classic.ts

```diff
@@ -0,0 +1,223 @@
+        };
+
+        if (this.debug) {
+          cmd.log('Executing web request...');
+          cmd.log(JSON.stringify(requestOptions));
+          cmd.log('');
+        }
+
+        return request.post(requestOptions);
+      })
+      .then((res: string): void => {
+        if (this.debug) {
+          cmd.log('Response:');
+          cmd.log(res);
+          cmd.log('');
+        }
+
+        const json: ClientSvcResponse = JSON.parse(res);
+        const response: ClientSvcResponseContents = json[0];
+        if (response.ErrorInfo) {
+          cmd.log(new CommandError(response.ErrorInfo.ErrorMessage));
+        }
+        else {
+          const sites: SPOSitePropertiesEnumerable = json[json.length - 1];
+          cmd.log(sites._Child_Items_.map(s => {
+            return {
+              Title: s.Title,
+              Url: s.Url
+            };
+          }).sort((a, b) => {
+            const urlA = a.Url.toUpperCase();
+            const urlB = b.Url.toUpperCase();
+            if (urlA < urlB) {
+              return -1;
+            }
+            if (urlA > urlB) {
+              return 1;
+            }
+
+            return 0;
+          }));
+        }
+        cb();
+      }, (err: any): void => this.handleRejectedPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-t, --type [type]',
+        description: 'type of classic sites to list. Allowed values are STS#0|BLOG#0|BDR#0|DEV#0|OFFILE#1|EHS#1|BICenterSite#0|SRCHCEN#0|BLANKINTERNET#0|BLANKINTERNETCONTAINER#0|ENTERWIKI#0|PROJECTSITE#0|PRODUCTCATALOG#0|COMMUNITY#0|COMMUNITYPORTAL#0|SRCHCENTERLITE#0|visprus#0|GROUP#0|SITEPAGEPUBLISHING#0.',
+        autocomplete: ['STS#0','BLOG#0','BDR#0','DEV#0','OFFILE#1','EHS#1','BICenterSite#0','SRCHCEN#0','BLANKINTERNET#0','BLANKINTERNETCONTAINER#0','ENTERWIKI#0','PROJECTSITE#0','PRODUCTCATALOG#0','COMMUNITY#0','COMMUNITYPORTAL#0','SRCHCENTERLITE#0','visprus#0','GROUP#0','SITEPAGEPUBLISHING#0']
+      },
+      {
+        option: '-f, --filter [filter]',
+        description: 'filter to apply when retrieving sites'
+      },
+      {
+        option: '--includeOneDriveSites [includeOneDriveSites]',
+        description: 'Set if you also want to retrieve OneDrive sites'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
```

Comment:
Let's remove this as we don't want to keep tracking which templates are available in SPO and updating the CLI each time there is a change